src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/sendorder/SendOrderApplicantDocumentHandler.java

```diff
@@ -96,9 +100,11 @@ private static void setConfidentialBulkPrintFieldForApplicant(FinremCaseDetails
     }
 
     protected void setConsolidateCollection(FinremCaseData caseData, List<ApprovedOrderCollection> orderCollection) {
+        List<ApprovedOrderCollection> orderCollectionCopy = documentHelper.deepCopyArray(orderCollection,
+            new TypeReference<List<ApprovedOrderCollection>>() {});
```

Comment:
why we need to make this copy?

Reply:
Here, we have multiple collection, If we don't copy then same object reference is use and if we apply category to one collection then it get refer in other collection